SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialActorUtils.h

```diff
@@ -2,18 +2,38 @@
 
 #pragma once
 
-#include "CoreMinimal.h"
+#include "EngineClasses/SpatialNetConnection.h"
 
 #include "Components/SceneComponent.h"
+#include "Containers/Array.h"
+#include "Containers/UnrealString.h"
 #include "Engine/EngineTypes.h"
-#include "EngineClasses/SpatialNetConnection.h"
 #include "GameFramework/Actor.h"
 #include "GameFramework/Controller.h"
 #include "GameFramework/PlayerController.h"
+#include "Math/Vector.h"
 
 namespace SpatialGDK
 {
 
+inline AActor* GetHierarchyRoot(const AActor* Actor)
```

Comment:
If this object is the root, should it return itself instead of null?

Reply:
Implementation detail, idm either way. Potentially might give less 1 or 2 less constant time looks in exchange for more bookkeeping